### PR TITLE
Update it.json to 2.0.3

### PR DESCRIPTION
**Pull request by fabioferrero90**

### CODE REVIEW

File: it.json [hunk 1]
```diff
@@ -1296,8 +1320,6 @@
                 "TITLE": "Vendi {coin}",
                 "DESCRIPTION": "Inserisci la quantità di {coin} (in $) che vuoi vendere",
                 "PLACEHOLDER": "$1000",
```
Comment (Maximus7474) [inline it.json:1322]: Syntax error, remove the last `,` from `"$1000",`

File: it.json [hunk 2]
```diff
@@ -144,6 +151,7 @@
             "CLOUD_BACKUP": "Backup Cloud",
             "YOUR_PROFILE": "Il tuo Profilo",
             "CREATE_BACKUP": "Crea Backup",
+            "BACKUPS": "Backup",
```
Comment (Maximus7474) [inline it.json:154]: Can be removed, duplicate entry with line 161.

**Review by Maximus7474 — CHANGES_REQUESTED** [review]
Thanks for your contribution !
Could you fix the two comments from my review ?

**Comment by RealSly90** [comment]
Done

**Comment by Maximus7474** [comment]
Thanks ! I will notify the devs

**Comment by Maximus7474** [comment]
Hey @fabioferrero90 

As your PR couldn't be merged previously, could you update your file with the newest en.json contents ?

**Comment by Maximus7474** [comment]
I will merge this PR, as part of an initiative of bringing all the locales up to date or as much as possible.

**Pull request merged**